xir/program.py

```diff
@@ -179,7 +196,7 @@ def use_floats(self) -> bool:
     @property
     def wires(self) -> Sequence[Wire]:
         """Returns the wires this observable statement is applied to."""
-        return tuple(wires for _, wires in self.terms)
+        return tuple(wires for _, wires in self.factors)
```

Comment:
Are you sure this is correct? Before, `self.terms` returned a sequence of pairs.
```suggestion
        return tuple(factor.wires for factor in self.factors)
```

Reply:
this was just a rename from terms to factors, so should function the same

Reply:
@puzzleshark but didn't the shape also change from a tuple of size 2 to a tuple of size 3?